### PR TITLE
Fix typos in "category" and unify capitals on "Back To Xyz List" buttons

### PR DESCRIPTION
**Pull request by drummer83**
#### What? Why?
Fixes two occurrences of the typo categeory (additional e).
Makes the labels on the "Back To Xyz List" buttons in the admin area use a capital letter uniformly. 

image

image

#### What should we test?

- Visit admin pages with a button "Back To Xyz List", e.g. edit product details, edit order, edit shipping methods, edit product category,  edit countries, etc.
- Check the label of the 'Back To Xyz List' button.

#### Release notes

Changelog Category (reviewers may add a label for the release notes):

- [x] User facing changes
- [ ] API changes (V0, V1, DFC or Webhook)
- [ ] Technical changes only
- [ ] Feature toggled

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by chahmedejaz — APPROVED** [review]
The button texts look so much better now. Thanks for taking a look at these 🙌

**Review by rioug — APPROVED** [review]
Nice one ! :+1:

**Comment by drummer83** [comment]
@filipefurtad0, @RachL Could any of you test this one please, when there's time? Thanks! :pray:

**Comment by RachL** [comment]
@drummer83 I see 2 occurrences missing :

customer details : 

image

and return authorizations :

image

do you want to handle them here or in a separate PR?

**Comment by RachL** [comment]
and just to be sure, is this the expected result for shipping method and payment methods? I'm surprise they have capital letters unlike "orders" and "products"

image

**Comment by RachL** [comment]
@drummer83 I'm unsure of the expected result wanted here. If the screenshots represent the expected results, it then means that the other tabs are wrong:

image

image

We should then have Back To Order List and Back To Product List here?

**Comment by drummer83** [comment]
Hi @RachL,
Thanks for having a look at this one!

The expected result is to have every word beginning with a capital letter: Back To Xyz List. This was the format which I found was used most, but not consistently. So I wanted to unify this.

It seems that the FR server is not using the updated language file, because also the typo is still there:
image

I think, this is because the FR server is using the en_FR.yml language file (which hasn't been updated yet) and not the fixed en.yml.
On the AU server, the updates can be seen, because the en_AU.yml file, which is used there, hasn't been updated for while. The updated strings don't even exist in the en_AU.yml file, so the software is falling back to the strings from en.yml - which are the updated ones. 

Long story short: 
You could test this on staging AU but I have also copied the content of en.yml to en_FR.yml, so the updates can be seen on the FR server too. However, I believe that this change will be gone after the next staging process on that server - not 100 % sure.

**Comment by RachL** [comment]
🤦‍♀️ yes silly me!! apologies, I should have thought of it as we just had UK prod down vs AU still up because AU uses the content of the code...

Merging! Thanks for this 💚
